File: samples/src/main/java/tk/zielony/carbonsamples/widget/ToolbarActivity.java
package tk.zielony.carbonsamples.widget;

import android.os.Bundle;
import android.view.Menu;
import android.view.MenuInflater;

import carbon.widget.Toolbar;
import tk.zielony.carbonsamples.R;
import tk.zielony.carbonsamples.SamplesActivity;

public class ToolbarActivity extends SamplesActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_toolbar);

        Toolbar toolbar = (Toolbar) findViewById(R.id.toolbar);
        toolbar.setIconVisible(true);

        setSupportActionBar(toolbar);
        setTitle("Toolbar");
    }

    @Override
    public boolean onCreateOptionsMenu(Menu menu) {
        MenuInflater inflater = getMenuInflater();
        inflater.inflate(R.menu.menu_navigation, menu);
        return true;
    }
}

File: samples/src/main/java/tk/zielony/carbonsamples/widget/ToolbarActivity.kt
package tk.zielony.carbonsamples.widget

import android.os.Bundle
import android.view.Menu
import kotlinx.android.synthetic.main.activity_toolbar.*
import tk.zielony.carbonsamples.ActivityAnnotation
import tk.zielony.carbonsamples.R
import tk.zielony.carbonsamples.SamplesActivity

@ActivityAnnotation(layout = R.layout.activity_toolbar)
class ToolbarActivity : SamplesActivity() {
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)

        toolbar.setIconVisible(true)

        setSupportActionBar(toolbar)
        title = "Toolbar"
    }

    override fun onCreateOptionsMenu(menu: Menu): Boolean {
        val inflater = menuInflater
        inflater.inflate(R.menu.menu_navigation, menu)
        return true
    }
}
